package-lock.json

```diff
@@ -1,8310 +1,114 @@
 {
   "name": "smartsheet",
-  "version": "3.1.4",
-  "lockfileVersion": 2,
+  "version": "3.1.5",
+  "lockfileVersion": 1,
```

Comment:
This PR is modifying the lockfile version currently used, which is _probably_ fine but would perhaps be safer done in a separate PR. If you re-run npm install using Node 14.17.3 it will preserve the existing lockfile format.

Perhaps we could consider adding the volta pin to package.json from my closed PR a little while ago?

```json
  "volta": {
    "node": "14.17.3"
  }
```

Reply:
The lockfile appears to have already been at version 2 so it must have been generated with a node version > 15. In that case to prevent us from going back to lockfile version 1 unintentionally should we volta pin to v16? What are your thoughts on this?

Reply:
Looks like that was inadvertently done during the package bump for the revert (3.1.4) https://github.com/smartsheet/smartsheet-javascript-sdk/commit/8675c13a8711ee7d343386a2c6e738d125283e7b

Kind of a tough call in this case. Like I said, it's _probably_ fine to leave it as is? Theoretically npm should be able to modify its own lockfile schema without any breaking changes to the library code. I do think we should pin to _something_ so this doesn't happen again. 16 would be a good choice, just a matter of how much extra risk to add to this PR. I assume the current lockfile was made with 16.x, so maybe we just consider ourselves upgraded and stick with that.

Reply:
Yeah I agree the safest option is likely to leave it at version 2 and pin to node 16 for future development. This will limit the `package-lock.json` changes to removing `request` and adding `axios`. Making this change now.